tests/network/hotplug.go

```diff
@@ -238,6 +238,28 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			Entry("In place", decorators.InPlaceHotplugNICs, inPlace),
 			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
 		)
+
+		DescribeTable("VMI status should report the MAC address specified in the VM template spec", func(plugMethod hotplugMethod) {
+			waitForSingleHotPlugIfaceOnVMISpec(hotPluggedVMI)
+			hotPluggedVMI = verifyDynamicInterfaceChange(hotPluggedVMI, plugMethod)
+
+			hotPluggedVM, err := kubevirt.Client().VirtualMachine(hotPluggedVM.Namespace).Get(context.Background(), hotPluggedVM.Name, &metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			Eventually(func(g Gomega) {
+				vmIfaceSpec := vmispec.LookupInterfaceByName(hotPluggedVM.Spec.Template.Spec.Domain.Devices.Interfaces, ifaceName)
+				g.Expect(vmIfaceSpec).NotTo(BeNil(), "VM spec should contain the new interface")
+
+				vmiIfaceStatus := vmispec.LookupInterfaceStatusByName(hotPluggedVMI.Status.Interfaces, ifaceName)
+				g.Expect(vmiIfaceStatus).NotTo(BeNil(), "VMI status should report the hotplugged interface")
+
+				g.Expect(vmiIfaceStatus.MAC).To(Equal(vmIfaceSpec.MacAddress),
+					"hot-plugged iface in VMI status should have a MAC address as specified in VM template spec")
+			}, time.Minute*2, time.Second*3).Should(Succeed())
```

Comment:
Waiting 2 minutes after the interface has been already been hotplug seems an overkill. 30sec should be more than enough, don't you think?